content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -38,6 +38,8 @@ plane and the sidecars for the Istio data plane.
 
 ## Installation steps
 
+The following commands has relative references in the Istio directory, and should be executed in the root of the Istio  directory.
+
```

Comment:
remove one space between `Istio` and `directory`

Reply:
done